ml-fundamentals/ml-fundamentals-checklist.md

```diff
@@ -0,0 +1,51 @@
+# ML Fundamentals checklist
+
+This checklist helps ensure that our ML projects meet our ML Fundamentals.
+
+## Data Quality and governance
+
+- [ ] There is access to data.
+- [ ] Labels exist for dataset of interest.
+- [ ] Data quality evaluation.
+- [ ] Able to track data lineage.
+- [ ] Understanding of where the data is coming from and any policies related to data access.
+
+## Feasability study
+
+- [ ] A feasability study was performed to assess if the data supports the proposed tasks.
+- [ ] Rigurous Exploratory data analysis was performed (including analysis of data distribution).
+- [ ] Hypotheses were tested producing sufficient evidence to either support or reject that an ML approach is feasible to solve the problem.
+- [ ] ROI estimation and risk analysis was performed for the project.
+- [ ] ML outputs/assets can be integrated within the production system.
+- [ ] Recommendations on how to proceed have been documented.
+
+## Evaluation and metrics
+
+- [ ] Clear definition of how performance will be measured.
+- [ ] The evaluation metrics are somewhat connected to the success criteria.
+- [ ] The metrics can be calculated with the datasets available.
+- [ ] Evaluation flow is applicable for all present and future models (i.e. does not assume some prediction structure or method-specific process).
+- [ ] Evaluation code is unit-tested and reviewed by all team members.
+- [ ] Evaluation flow facilitates further results and error analysis.
+
+## Model Baseline
+
+- [ ] Well defined baseline model exists and its performance is calculated.

```

Comment:
"Well defined" is a little hard to define ;) maybe clearly documented might make it more understandable?

Reply:
Added link to the section on the ml-model-checklist where there are more details on what is meant by a well defined baseline. Do you think this makes it clearer?